cmd/delete.go

```diff
@@ -112,7 +112,7 @@ func (cmd *DeleteCmd) Run(ctx context.Context, devPodConfig *config.Config, args
 		GracePeriod: duration,
 	})
 	if err != nil {
-		return err
+		return errors.Wrap(err, clientimplementation.DeleteWorkspaceFolder(client.Context(), client.Workspace(), log.Default).Error())
```

Comment:
I'm not sure this is a good idea, we have the `--force` flag to make sure even if the deletion fails the workspace is deleted.

Reply:
This was not working with the `--force` flag, even with it, it was stopping at this very point